package.json

```diff
@@ -46,15 +46,18 @@
     "homepage": "https://github.com/textkernel/oneui#readme",
     "dependencies": {
         "@babel/polyfill": "7.0.0",
-        "@react-google-maps/api": "1.4.2",
+        "@react-google-maps/api": "1.5.1",
         "css-vars-ponyfill": "1.16.4",
-        "prop-types": "^15.7.2",
-        "rc-slider": "8.6.13",
         "downshift": "3.2.10",
         "fast-memoize": "2.5.1",
+        "prop-types": "^15.7.2",
+        "rc-slider": "8.6.13",
         "react-modal": "3.9.1"
     },
     "peerDependencies": {
+        "@fortawesome/fontawesome-svg-core": "^1.2.21",
+        "@fortawesome/free-solid-svg-icons": "^5.10.1",
+        "@fortawesome/react-fontawesome": "^0.1.4",
```

Comment:
Alex recommended me to use in NextGent UI `react-icons`. For it doesn't need to add 3 dependencies and it easy to use.

Reply:
ok, I'll have a look